HHS/Head-Start-TTADP
# [TTAHUB-2146] Queue rate limiter

## Description of change

The issue this attempts to fix is overloading the queue with retries. Based on @GarrettEHill initial investigation we were restarting worker jobs before the previous job had time to clean up its memory.

I tested this on dev for 6 month's (1515 resources) I didn't see the memory of the app spike above 220 MB. I will need to test this again for a yr once its on staging.

## How to test

- Review both the documentation on BULL (https://optimalbits.github.io/bull/) and the code change
- Make sure you agree with the change and settings.
- Triggering ES events like submitting or deleting a report can also be tested on DEV.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
